integration/nwo/token/common/ppmgen.go

```diff
@@ -172,16 +185,20 @@ func (d *DLogPublicParamsGenerator) Generate(tms *topology.TMS, wallets *topolog
 		}
 		for _, issuer := range wallets.Issuers {
 			// Build an MSP Identity
-			provider, _, err := x509.NewKeyManager(issuer.Path, "", msp2.AuditorMSPID, nil, issuer.Opts)
+			km, _, err := x509.NewKeyManager(issuer.Path, "", msp2.AuditorMSPID, nil, issuer.Opts)
 			if err != nil {
-				return nil, errors.WithMessage(err, "failed to create x509 provider")
+				return nil, errors.WithMessage(err, "failed to create x509 km")
 			}
-			id, _, err := provider.Identity(nil)
+			id, _, err := km.Identity(nil)
 			if err != nil {
 				return nil, errors.WithMessage(err, "failed to get identity")
 			}
 			if tms.Issuers[0] == issuer.ID {
```

Comment:
Use `GetUnique`, `GetAny` or `GetFirst` instead of `tms.Issuers[0]` to make your intent clear.